pkg/tests/path_test.bzl

```diff
@@ -0,0 +1,86 @@
+# Copyright 2021 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for path.bzl"""
+
+load("@bazel_skylib//lib:unittest.bzl", "analysistest", "asserts", "unittest")
+load("//:mappings.bzl", "pkg_mkdirs")
+load("//:path.bzl", "compute_data_path")
+
+##########
+# Test compute_data_path
+##########
+def _compute_data_path_test_impl(ctx):
+    env = analysistest.begin(ctx)
+    target_under_test = analysistest.target_under_test(env)
+    asserts.equals(
+        env,
+        ctx.attr.expected_path,
+        compute_data_path(ctx, ctx.attr.in_path),
+    )
+    return analysistest.end(env)
+
+compute_data_path_test = analysistest.make(
+    _compute_data_path_test_impl,
+    attrs = {
+        "in_path": attr.string(mandatory = True),
+        "expected_path": attr.string(mandatory = True),
+    },
+)
+
+def _test_compute_data_path():
+    pkg_mkdirs(
+        name = "dummy",
+        dirs = [],
+        tags = ["manual"],
+    )
```

Comment:
Is this strictly necessary?  If it is, it might be worth a comment saying it's unused and something needs to be passed into `target_under_test`.